README.md

```diff
@@ -32,4 +32,7 @@ To upload the Privacy Preferences Policy Control Payload to Jamf Pro 10.7.0 and
 the profile will need to be signed before uploading.
 ![alt text](/Images/UploadSigned.png "Upload signed")
 
+## Importing
+Signed and unsigned profiles can be imported.
 
+![alt text](/Images/ImportProfile.png "Import profiles")
```

Comment:
All of the "alt text" inside the square brackets in front of the image path (six of them in this file) are the text that is displayed for Markdown viewers that can't display inline images.  Having them all say literally "alt text" is not helpful for screen readers or other Markdown viewers without images.

This one, for example, should maybe be something like

`+ ![Import any profile](/Images/ImportProfile.png "Import profiles")`

Reply:
done